locales/en.default.json

```diff
@@ -359,6 +359,11 @@
       "remove_all_items_confirmation": "Remove all {{ quantity }} items from your cart?",
       "remove_all": "Remove all",
       "cancel": "Cancel"
+    },
+    "quick_order_form": {
+      "search": "Search by product, variant or SKU",
+      "search_number_items": "{{ count }} items",
```

Comment:
@melissaperreault @eugenekasimov Im not sure if `line total` is the right language 👀 Each line is more of the variant total 

Line total is what we use in cart where discounts are applied etc. Here they arent. That may lead to confusion. Should we use something similar to quick order form?

Reply:
Could it just be `total`?

Reply:
Related question re: `Product subtotal` at the bottom. My understanding is that the form can list more than one product - in this case, should that just say `Subtotal`?